src/TokenKind.php

```diff
@@ -87,7 +87,7 @@ class TokenKind {
     const YieldFromKeyword = 167;
     const FnKeyword = 168;
     const MatchKeyword = 169;
-    const IterableKeyword = 170;
+    /** @deprecated use IterableReservedWord */const IterableKeyword = 170;
```

Comment:
```suggestion
    /** @deprecated use IterableReservedWord */
    const IterableKeyword = self::IterableReservedWord;
```

My earlier suggestion was shorthand - style guidelines I'm aware of for php have doc comments on separate lines.
Single-line comments are used for properties when the comment is just one line.